process_submissions.R

```diff
@@ -0,0 +1,5 @@
+## Send from submissions to forecasts
+minioclient::mc_alias_set("efi", "data.ecoforecast.org", "", "")
+minioclient::mc_cp(from = paste0("efi/spat4cast-submissions/", theme),
+                   to = paste0("efi/spat4cast-forecasts/", theme),
```

Comment:
hmm -- maybe we want `mc_mv()` instead?  Otherwise this command will get longer and longer as we accumulate copies in both places (if we want to do so we  could use `mc_mirror()` to avoid copying things that were sync'd earlier).  

Also minor note on paths, this will surprisingly duplicate the theme path in the dest folder, `spat4cast/lai_recovery/lai_recovery`.  Use a trailing slash after the name in the source to indicate that you want to copy everything in the folder but not the folder itself I think?